core/src/main/java/com/yahoo/imapnio/async/request/AbstractFolderActionCommand.java

```diff
@@ -16,7 +16,7 @@
 abstract class AbstractFolderActionCommand extends ImapRequestAdapter {
 
     /** Byte array for CR and LF, keeping the array local so it cannot be modified by others. */
-    private static final byte[] CRLF_B = { '\r', '\n' };
+    protected static final byte[] CRLF_B = { '\r', '\n' };
```

Comment:
byte array is mutable, Vikram has commented in my PR on my PR, so i keep it private within class.